recipes/default.rb

```diff
@@ -19,12 +19,17 @@
 # limitations under the License.
 #
 
-include_recipe('os-hardening::packages')
-include_recipe('os-hardening::limits')
-include_recipe('os-hardening::login_defs')
-include_recipe('os-hardening::minimize_access')
-include_recipe('os-hardening::pam')
-include_recipe('os-hardening::profile')
-include_recipe('os-hardening::securetty')
-include_recipe('os-hardening::suid_sgid') if node['os-hardening']['security']['suid_sgid']['enforce']
-include_recipe('os-hardening::sysctl')
+case node['platform_family']
+when 'debian', 'rhel'
```

Comment:
@grdnrio in my eyes a such thing might lead to confusion: as a cookbook user I usually do not expect such logic in the default recipe of included cookbook, I expect that the included code gets executed. If such `if` is required, it belongs in my eyes to some wrapper cookbook but not here